lalrpop/src/normalize/token_check/mod.rs

```diff
@@ -56,13 +87,16 @@ struct Validator<'grammar> {
     grammar: &'grammar Grammar,
     all_literals: Map<TerminalLiteral, Span>,
     conversions: Option<Set<TerminalString>>,
+    match_mappings: Option<Map<TerminalString, TerminalString>>,
```

Comment:
Similarly to before, it seems like we might as well put some comments here showing what these maps contain. Bonus points for an example.